apps/aecore/lib/aecore/tx/signed_tx.ex

```diff
@@ -253,7 +253,7 @@ defmodule Aecore.Tx.SignedTx do
   def encode_to_list(%SignedTx{} = tx) do
     [
       :binary.encode_unsigned(@version),
-      tx.signatures,
+      Enum.sort(tx.signatures),
```

Comment:
This change will break our signature verification code as our code currently assumes that the signatures are not sorted and have a 1-1 correspondence to sender's pubkeys.
If we serialize and then deserialize the SignedTx then the signature verification will fail when multiple signatures are present. The signatures also need to be sorted for the JSON api.
Here is the code to resolve issue #568 (the code misses typespecs and docs - I will add them soon): https://github.com/aeternity/elixir-node/commit/99a2f367b4254359a095f1dd86c9ebe1278ee3a0

Reply:
We will remove sorting function and we will leave everything here as it was. So then you can resolve it in your PR.